hackclub.com.yaml

```diff
@@ -1662,9 +1662,30 @@ nust:
     type: CNAME
     value: nusthackclub.netlify.app.
 ocsa:
-  - ttl: 1
+  - ttl: 3600
     type: TXT
-    value: google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+    values: 
+    - google-site-verification=tk_snQuv19iUwoOlsbKZv2Wwg5stfcusEm1Q8NxDiVA
+    - v=spf1 include:_spf.google.com ~all
+ 
+  - ttl: 3600
+    type: MX
+    values:
+      - exchange: aspmx.l.google.com.
+        preference: 1
+      - exchange: alt1.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt2.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt3.aspmx.l.google.com.
+        preference: 10
+      - exchange: alt4.aspmx.l.google.com.
+        preference: 10
+  - ttl: 1
+    type: ALIAS
+    value: uhegg.github.io.
```

Comment:
Is this the correct GitHub pages URL? There doesn't seem to be a repo at uhegg/uhegg.github.io nor is there a site at uhegg.github.io

Reply:
the websites here: https://uhegg.github.io/hackclub, but i beleive github doesn't want you to include anything after the slash